store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala

```diff
@@ -49,14 +49,6 @@ class PGSecondary(val config: Config) extends Secondary[SoQLType, SoQLValue] wit
     logger.info("shut down {} {}", dsConfig.host, dsConfig.database)
   }
 
-  // Return true to get all the events from the stream of updates from the data-coordinator
```

Comment:
As long as Robert is OK with this.  There was a ES secondary that did not want working copy.